src/app/services/member-service.ts

```diff
@@ -0,0 +1,5 @@
+export class MemberService {
```

Comment:
We should make this implement an interface and inject the interface instead of the implementation.